### ISSUE TITLE
Radio Buttons not announced by Screen Reader when selecting

### ISSUE THREAD

**Issue by jmwmusic** [musescore/MuseScore#20183]
### Issue type

UI bug (incorrect info or interface appearance)

### Bug description

In many places there are radio buttons which can be selected (checked) by pressing Space. Doing so does not announce the changed state until the button is refocused.

This may be partly because radio buttons are normally selected upon focus, not upon pressing Space when in NVDA edit mode.  In browse mode they can be skimed over to hear without selecting.  The browse/edit mode doesn't seem to be changeable in MuseScore so the assumption is it is acting in Edit mode.

There are various cases, below is just one:

### Steps to reproduce

1. From the File menu, choose New (or press Ctrl+N).
The New Score Wizard appears.
2. Press Arrow Down to choose Create from Template.
Notice the radio button remains unselected (unchecked) at this stage. Note: Normal Windows keyboard interface would be that the radio button is automatically selected.
3. Press Space to select.

Expected behaviour: the speech should announce "checked" as the radio button is now checked.

Actual behaviour is silence.
On the braille display, the radio button continues to be shown unchecked.
It is not until the control is refocused (by Arrow Up/Down) that the correct state is announced and displayed.

Note: Similar behaviour can be observed in several other places, e.g. the toolbar with Home, Score and Publish.

### MuseScore Version

4.1

### Regression

I don't know

### Operating system

Windows 10 with NVDA

**Comment by jmwmusic** [comment]
This seems to me to be part of the same issue with tabs and radio buttons, but if it isn't I can shift it to a separate issue:

In the New Score wizard, sub-dialogs for Time signature, it is possible to press Tab and land on unselected options. The norm is that in a group of radio buttons, the selected item in a group always receives focus when pressing Tab.

Steps:
1. From the File menu select New, or press Ctrl+N.
The New Score wizard appears.
2. Choose instruments or select a template, then press Next to move to the second page.
3. In the second page, press the Time signature button to open the sub-dialog.
The Time signature sub-dialog appears.
4. Tab around to confirm default behaviour: 4/4 radio button checked, and the options for the numeric time signatures.
5. From the 4/4 radio button, Arrow Down to Common time and press Space.
The Common time radio button is now selected - see above as this is not reflected by speech or braille.
6. Now press Tab.

Expected behaviour, nothing

Actual behaviour: Focus returns to the unselected 4/4 radio button and further pressing Tab does nothing.

This gives the impression that the user is now stuck, potentially causing confusion.

**Comment by msviolafangirl** [comment]
As an NVDA user, I can confirm this is an issue. I actually didn't pay much attention to it, so thanks for raising it.
